client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/index.tsx

```diff
@@ -91,6 +91,11 @@ const Launchpad: Step = ( { navigation, flow }: LaunchpadProps ) => {
 		}
 	}, [ siteSlug, site ] );
 
+	if ( launchpadScreenOption === 'skipped' ) {
+		window.location.assign( `/home/${ siteSlug }` );
```

Comment:
I don't think we should be calling `redirectToSiteHome()`, but that may be the right thing to do here.